### DISCUSSION TITLE
Rename META-INF/services

### DISCUSSION THREAD

**Discussion by Antus8** [eclipse-rdf4j/rdf4j#3576] [Q&A]
I'm creating an application for AppInventor that uses RDF4J. I import into Appinventor the jar file of my library but the problem is that the tool that creates the apk file ignores the META-INF/services folder, so the application doesn't work.
Is there a way to rename META-INF folder in order to let it be included in the apk and still work with my application? I don't know whether I rename it  the Java Service Provider Interfaces will still work.

**Comment by hmottestad** [comment]
This is a question best posed to the OpenJDK team or someone at Oracle. 

Otherwise see: https://github.com/eclipse/rdf4j/discussions/3554
